src/Assignment-3/assignment3.css

```diff
@@ -0,0 +1,72 @@
+/* Our CSS file. */
+
+.container,
+.content {
+    width: 100%;
+    height: 100%;
+}
+
+.div-container {
+    background-color: #f2f2f2;
+    margin: 5%;
+    height: 500px;
+}
+
+.div-container section {
+    display: inline-block;
+}
+
+.sec1 {
+    width: 40%;
```

Comment:
iFrame should take the full width of the right container

Reply:
okay I'll make it correct